lib/ruby-osc.rb

```diff
@@ -25,10 +25,11 @@ def to_osc_type
   end
 
   def self.coerce_argument arg
+    int = RUBY_VERSION >= "2.4" ? Integer : Fixnum
     case arg
     when OSCArgument then arg.to_osc_type
     when Symbol      then arg.to_s
-    when String, Float, Fixnum, Blob, String then arg # Pure osc 1.0 specification
+    when String, Float, int, Blob, String then arg # Pure osc 1.0 specification
```

Comment:
@nicb it should be enough to replace Fixnum with Integer, that should work also with Ruby < 2.4 since Integer used to be ancestor of Fixnum.

```
irb(main):003:0> Fixnum.ancestors
=> [Fixnum, Integer, Numeric, Comparable, Object, Kernel, BasicObject]
```

No need to check Ruby version